vector/src/main/res/drawable/splash.xml

```diff
@@ -4,14 +4,33 @@
 <layer-list xmlns:android="http://schemas.android.com/apk/res/android"
     android:opacity="opaque">
 
-    <!-- Background color -->
-    <item android:drawable="@color/riotx_accent" />
-
-    <!-- Centered logo -->
+    <!-- Background image -->
     <item>
-        <bitmap
-            android:gravity="center"
-            android:src="@drawable/riot_splash_white" />
+        <bitmap android:src="@drawable/bg_tchap_splashscreen" />
     </item>
 
+    <!-- Centered logo -->
+    <item
+        android:bottom="300dp"
+        android:drawable="@drawable/ic_tchap_logo"
+        android:gravity="center" />
+
+    <!-- Centered title -->
+    <item
+        android:bottom="120dp"
+        android:drawable="@drawable/ic_tchap_title"
+        android:gravity="center" />
+
+    <!-- Centered description -->
+    <item
+        android:bottom="40dp"
+        android:drawable="@drawable/ic_tchap_splashscreen_desc"
```

Comment:
I would expect a TextView here, we will talk about it in direct

Reply:
drawable is required here because of the layer-list, this last one is the well adapted to handle the splash screen